Jorge-Taylor.txt

```diff
@@ -0,0 +1,192 @@
+        Id is a unique selector that can only be applied to at most one per element.
+
+    10. What is a class? 
+        A class is another selector. another attribute.
+
+    11. What elements can I add a class to? 
+         All.
+
+    12. How many times can I use the same class on a page? 
+
+     As many times as you want but when it's time to target one it will target ALL.Unless
+there's multiple classes or an ID.
+
+    13. How do I get my link to open in a new tab?
+              _blank
+
+    14. What is the alt attribute used for? 
+         alt is used to specify alt text when like the href site is down or something isn't rendering.
+
+    15. How do I reference an id?
+               #
+
+    16. What is the difference between a section and a div               
+       well a div has no meaning its just a container or DIVider while a section has meaning and signifies a section.
+
+    17. What is CSS used for? 
+        styling your HTML.
+
+    18. How to we select an element? Example - every h2 on the page
+       h2{
+      }
+
+    19. What is the difference between a class and an id? - Give me an example of when I might use each one
+                  the only difference is that iD is very specific and unique and only usable once per element, also iDs have greater spesifity and trumps a class. while a class is not unique and can be used as many times as needed. iDs are like anchors and shouldn't be used for styling.
+<ul id="somethingUni"></ul>   <a class="className"></a>
+
+    20. How do we select classes in CSS?
+               .className
+
+
+    21. How do we select a p element with a single class of “human””?
+                 p.human
+
+    22. What is a parent child selector? When would this be useful? 
+           
+         ("parent > child") its used to select all children of the parent.
+          /*replacing "parent" with the parent element to be selected 
+          and "child" with the child element.*/
+
+
+    23. How do you select all links within a div with the class of sidebar?
+    .sidebar a:link
+          
+
+    24. What is a pseudo selector?
+            It is used to define certain state of an element.
+            They will have a colon before them.
+            (I could be wrong but I personally think of them like Methods in js since 
+            they I guess have like a function inside them theres no need to declare or put
+            anything inside them as they work as is. like :hover :link :visited once added to your code 
+            the computers/browser already knows what to do when it sees them. (Again I could be wrong for sure lol) 
+
+
+    25. What do we use the change the spacing between lines?
+        Breaks? <br> or do you mean like "%20"?
+
+    26. What do we use to change the spacing between letters?
+            using CSS? letter-spacing:
+
+
+    27. What do we use to to change everything to CAPITALS? lowercase? Capitalize?
+                          Capitalize
+
```

Comment:
What is the CSS property you need to use though, capitalize is right, but what comes on the left side? -blank-:capitalize